accel_code/utils.py

```diff
@@ -114,6 +115,14 @@ def get_test_data_path(project):
             '3-Experiment',
```

Comment:
the function [get_test_data_path]9https://github.com/HBClab/accelBIDSTransform/pull/8/files#diff-77d1e897c73caa48a6bd94de0d728393R1080 should get a `root_directory` parameter so we can define where the projects are relative to where the server is mounted.